wrapspawner/wrapspawner.py

```diff
@@ -129,6 +129,14 @@ def poll(self):
         else:
             return _yield_val(1)
 
+    if hasattr(Spawner, 'progress'):
+        @property
+        def progress(self):
+            if self.child_spawner:
+                return self.child_spawner.progress
+            else:
+                return self.progress
```

Comment:
I think this `else` will hit a recursion limit. Instead, it should probably raise an error saying that it can't get progress when the child spawner doesn't exist.